src/main/java/io/servertap/PluginEntrypoint.java

```diff
@@ -71,8 +71,11 @@ public void onEnable() {
                 // Player routes
                 get("players", PlayerApi::playersGet);
                 get("players/all", PlayerApi::offlinePlayersGet);
+                get("players/op", PlayerApi::getOps);
                 get("players/:uuid", PlayerApi::playerGet);
                 get("players/:playerUuid/:worldUuid/inventory", PlayerApi::getPlayerInv);
+                post("players/:playerUuid/op", PlayerApi::opPlayer);
+                post("players/:playerUuid/deop", PlayerApi::deopPlayer);
```

Comment:
Can't these be combined into one endpoint, utilizing a `POST` request to set op status with a query param `value=false` / `value=true`? I feel that two endpoints serving identical but flipped purposes doesn't make much sense.

Reply:
They definitely can be, it was done to make the routes more concise. What do you think @phybros?

Reply:
To be properly resty we could POST or DELETE from an “ops” collection, I.e. `/server/ops` with a formParam for playerUuid or username

Reply:
I'll update it accordingly.